### PR TITLE
Fix input value print over control buttons

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Expand the dialog dynamically to allow more input
and fix overprinting to the control buttons.

Signed-off-by: Timothy Asir <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-02-03 13-51-08

Dialog expanding
Screenshot from 2023-02-03 13-52-09

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar please review

**Comment by SanjalKatiyar** [comment]
doesn't look entirely correct to me...  in the attached screenshots I can't see border of the input box...

File: packages/shared/modals/Selector.tsx [hunk 1]
```diff
@@ -282,7 +282,7 @@ export class SelectorInput extends React.Component<
     };
 
     return (
-      <div className="co-search-input pf-c-form-control">
+      <div className="co-search-input">
```
Comment (SanjalKatiyar) [inline packages/shared/modals/Selector.tsx:285]: this is not correct, it will remove the border of the input...

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-02-06 14-03-39
Screenshot from 2023-02-06 14-00-13

**Comment by SanjalKatiyar** [comment]
Fixes: https://github.com/red-hat-storage/odf-console/issues/623

**Pull request merged**